data-jdbc/src/main/java/io/micronaut/data/jdbc/config/DataJdbcConfiguration.java

```diff
@@ -79,6 +80,21 @@ public void setSchemaGenerate(SchemaGenerate schemaGenerate) {
         }
     }
 
+    /**
+     * @return an indicator telling whether foreign key constraints should be created when creating schema or dropped when dropping tables
+     */
+    public boolean isHandleForeignKeys() {
```

Comment:
This flag might not make sense, only added in case of bugs so users can disable it